at_secondary/at_secondary_server/lib/src/notification/stats_notification_service.dart

```diff
@@ -79,15 +80,7 @@ class StatsNotificationService {
       // Iterates on the list of active connections.
       await Future.forEach(connectionsList,
           (InboundConnection connection) async {
-        // If a monitor connection is stale for 15 seconds,
```

Comment:
Removing this block entirely will mean that stats are written not only by the scheduled send which runs every `AtSecondaryConfig.statsNotificationJobTimeInterval` seconds, but also every time there is a Change operation (see ChangeVerbHandler) or a notify that results in a keyStore change operation (see NotifyVerbHandler).

Is that what we want, or do we just want it to run periodically?

Reply:
@gkc : Added _lastSentTime to StatsNotificationService which will prevent notification on every update and notify.

Reply:
OK so that will match current behaviour pretty much, I think this is good enough for now given we will have streaming sync in the near future